changelog

```diff
@@ -1,9 +1,11 @@
-turnkey-torrentserver-17.0 (1) turnkey; urgency=low
+turnkey-torrentserver-17.1 (1) turnkey; urgency=low
 
+  * Updating torrentserver and its dependencies to tkldev v17. Updated libinithooks dialog_wrapper.
+ 
   * Note: Please refer to turnkey-core's 17.0 changelog for changes common to all
-    appliances. Here we only describe changes specific to this appliance.
+    appliances. Here we only describe changes specific to this appliance. 
```

Comment:
Not that big a deal, but it looks like you've accidentally added a trailing space to the end of this line.

Reply:
Went through and removed this trailing space from the documentation.